03/app.test.js

```diff
@@ -1 +1,23 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+test('returns 1 when the range is from 1 to 1', () => {
+    const showNumber = randomNumber(1, 1);
+    expect(showNumber).toBe(1);
+});
+
+test('return a radnom number in the range', () => {
+    const min = 3
+    const max = 4
+    const number = randomNumber(min, max)
+
+    expect(number).toBe(4)
+
+})
+
+test('throws an error when argument is not a number', () => {
+    function generateRandomItem() {
+        randomNumber("5");
+    }
+
+    expect(generateRandomItem).toThrow();
```

Comment:
Zawsze warto wpisać komunikat jakiego się spodziewamy - inaczej nie wiemy czy to błąd, którego się spodziewamy tj. `expect(generateRandomItem).toThrow('message');`